### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brandonroots**
Submitting constrained writing exercise

### CODE REVIEW

File: 01_intro/README.md [hunk 1]
```diff
@@ -47,3 +47,5 @@
 ## Add your name / link to your assignment below
 *(Please note you are welcome to post under a pseudonym and/or password protect your published assignment. Here is some [helpful information on privacy options for an NYU blog](https://nyu.service-now.com/sp?id=kb_article&sysparm_article=KB0012245&sys_kb_id=b2ddc9da004aa1002a5d036a271e5f70&spa=1). Finally, if you prefer not to post your assignment at all here, you may email the submission.)*
 * Name -- [assignment title](assignment url)
+
+Brandon Roots -- RhymeTime (http://brandonroots.com/itp/fall2020/rhymetime/rhymetime.html)
```
Comment (shiffman) [inline 01_intro/README.md:51]:
```suggestion
Brandon Roots -- [RhymeTime](http://brandonroots.com/itp/fall2020/rhymetime/rhymetime.html)
```

Great work! Minor suggestion above  markdown syntax for a link is typically `[link name](url)`. Try seeing if it works for you to accept this change then I will merge!

**Review by shiffman — CHANGES_REQUESTED** [review]
One small change requested!

**Comment by brandonroots** [comment]
Yes! Sorry, am new to using GitHub. Believe I have accepted the change...

**Review by shiffman — APPROVED** [review]
Perfect! No need to apologize, everyone is new and this is a friendly, learning space for us to all try figuring out this whole git and GitHub thing!

**Pull request merged**